src/components/InputWithLabel/inputWithLabel.tmpl.js

```diff
@@ -0,0 +1,16 @@
+export const template = `
+	<div class="inner">
+		<label 
+			class="label {% labelClassName %}" 
+			for="{% labelId %}" 
+		>
+			{% labelText %}
+		</label>
+		<input 
+			class="input {% className %}" 
+			{% attributes %} 
+			name="{% name %}" 
+			{% value %} 
```

Comment:
А почему по-разному написаны атрибуты? value как-то по-другому обрабатывается шаблонизатором? Стоит сохранять единообразие, это хорошо скажется на чтении и поддержке кода
```suggestion
			name="{% name %}" 
			value="{% value %}"
```